src/components/Stations/AddNew.js

```diff
@@ -3,11 +3,11 @@
 import React from "react";
 import { Link } from "react-router-dom";
 
-export default class AddNewStation extends React.Component {
+export default class AddNew extends React.Component {
```

Comment:
Again, I'm being pathetically pedantic. (Sorry.) I think having two components called `AddNew`, even though they're in separate folders, could be confusing and perhaps isn't best practice when they are quite different things. Would it be possible to go back to `AddNewCase` and `AddNewStation`?

Reply:
I think there are a few things going on here: 
1. Naming of the variables and components in the Case and Station folders specifically is till ongoing. The words "Stations" and "Cases" were confusing to me on Friday (and I assume would be to any future dev) so I would suggest exchanging these to "Tiles" and "Banners" which holds information not only of what the elements are but also how they look 

2. The duplicate naming in these folders came from me starting to identify opportunities for refactoring (so see this as an attempt of an suggestion) and the naming does not have to stay duplicate, but i'd like to discuss refactoring at some point 

3. To answer your question we can call them "AddNewTile" and "AddNewBanner", but I would like to open up the discussion to possibly refactor these (it depends a bit on what the back looks like)

4. finally, any set names should be captured in the anatomy figma thingy (I suggest we stick an image of it in the wiki when we hand over)